### PR TITLE
Add CONTRIBUTION guide

### PR DESCRIPTION
**Pull request by gribok**
Based on #169, provides first version of contrib guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+* [Enhancement issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/enhancement) - issues for new feature or request
+* [Help wanted issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/help%20wanted) -  extra attention is needed
+* [Documentation issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/documentation) - improvements or additions to documentation
+
+## Styleguide
+
+Please follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guide.
+See [Running Test](./README.md#running-tests) and [Linting](./README.md#linting) for further instructions.
+
```
Comment (Kilo59) [inline CONTRIBUTING.md]: Should we add a step here about running the formatter?
`pipenv run fmt` or `make fmt` or `black . -l 120`

Comment (gribok) [reply CONTRIBUTING.md]: Thanks for feedback. Added `make ftm` to styleguide section.

**Comment by gribok** [comment]
Some points to discuss? :-)

File: .travis.yml [hunk 2]
```diff
@@ -6,5 +6,8 @@ python:
 install:
   - "pip install pipenv"
   - "pipenv install --dev --skip-lock"
+  - "pipenv install coveralls"
```
Comment (Kilo59) [inline .travis.yml]: IMO this coveralls stuff should be a totally separate PR but...

You shouldn't use `pipenv` install this dependency here.
1. Either add it to the `Pipfile` so that it gets installed along with the `dev` dependencies
2. install it directly with pip in the first install step. `pip install pipenv coveralls`

File: Pipfile [hunk 3]
```diff
@@ -8,7 +8,9 @@ bandit = "*"
 black = "==19.10b0"
 isort = "*"
 pytest = "*"
+pytest-cov="*"
 pylint = "*"
+coveralls = "*"
```
Comment (Kilo59) [inline Pipfile]: Updates to the `Pipfile` should always be accompanied by a new `Pipfile.lock`
Run `pipenv lock` to regenerate a new lockfile.

**Comment by Kilo59** [comment]
@gribok Can I remove the coveralls stuff from this branch and merge it?

**Comment by gribok** [comment]
I am going to close the PR, seperate the changes for #169 and #207 in two branches and create new PRs based on mentioned issues.
